dashboard/src/utils/helper.js

```diff
@@ -5,6 +5,13 @@ export const uid = () => {
   return head + tail;
 };
 
+/**
+ * Check if password entered by user is valid or not
+ * @function
+ * @param {string} password - Entered email
+ * @param {number} passwordLength - Length of Password
```

Comment:
The parameter is badly named -- it should be `minimumLength` -- so your description needs to make up for that!